gno.land/pkg/integration/testing_integration.go

```diff
@@ -270,7 +282,65 @@ func setupGnolandTestScript(t *testing.T, txtarDir string) testscript.Params {
 					ts.Fatalf("error creating account %s: %s", args[0], err)
 				}
 
-				newUserBalances = append(newUserBalances, balance)
+				// Add balance to genesis
+				genesis := ts.Value(envKeyGenesis).(*gnoland.GnoGenesisState)
+				genesis.Balances = append(genesis.Balances, balance)
+			},
+			// use load a specific packagae from the example folder or from the working directory
+			"use": func(ts *testscript.TestScript, neg bool, args []string) {
+				creator := crypto.MustAddressFromString(DefaultAccount_Address) // test1
+				defaultFee := std.NewFee(50000, std.MustParseCoin("1000000ugnot"))
+
+				// special dirs
+				workDir := ts.Getenv("WORK")
+				examplesDir := filepath.Join(gnoRootDir, "examples")
+
+				var txs []std.Tx
+				switch len(args) {
+				case 1:
+					var path string
+					if args[0] == "all" {
+						// if `all` is specified fully load example folder
+						path = examplesDir
+					} else {
+						path = filepath.Clean(args[0])
+						if !strings.HasPrefix(path, workDir) {
+							path = filepath.Join(examplesDir, path)
+						}
+					}
+
+					var err error
+					txs, err = gnoland.LoadPackagesFromDir(path, creator, defaultFee, nil)
+					if err != nil {
+						ts.Fatalf("`use` unable to load package(s) %s: %s", path, err)
+					}
+				case 2:
+					path := filepath.Clean(args[0])
+					if !strings.HasPrefix(path, workDir) {
+						path = filepath.Join(examplesDir, path)
+					}
+
+					pkg := gnomod.Pkg{
+						Dir:      path,
+						Name:     filepath.Clean(args[1]),
+						Draft:    false,
```

Comment:
I think `Draft` and `Requires` are unnecessary here and can be dropped for brevity.